applications/GeoMechanicsApplication/tests/test_element_lab/oedometer_ULFEM/README.md

```diff
@@ -0,0 +1,30 @@
+# Oedometer ULFEM test
+
+This test simulates an oedometer lab test. The UPwUpdatedLagrangianElement2D3N element is used to simulate the soil
+sample in a very simple 4-noded mesh, as displayed in the following figure:
+
+![MeshStructure](MeshStructure.svg)
+
+## Setup
+
+The test is performed in a single stage, with the following conditions:
+
+-   Constraints:
+    -   The X and Y displacement in the bottom two nodes (3 and 4) are fixed.
+    -   The X displacement in all nodes is fixed.
+-   Material:
+    -   The material is described using a linear elastic material with a GeoLinearElasticPlaneStrain2DLaw.
+-   Conditions:
+    -   A UPwFaceLoadCondition2D2N is added to the top of the cube (nodes 1 and 2).
+-   Loads:
+    -   A line load in the y direction is applied to the top of the cube (nodes 1 and 2), linearly ramping up from 0 to
+      -1000000 kPa in the time interval \[0, 1\].
+
+## Assertions
+
+In this test, there are two results checks:
+
+1. It is asserted that the effective stress is -1000000 kPa in the integration points of both elements in the y
+   direction and 0.0 in the other directions, due to the Poisson ration being 0.0.
+2. It is asserted that the displacement of one of the top nodes (1) is close to -0.0909090909516868 as a result of the
+   applied line load.
```

Comment:
Here you could add the displacement calculation due to Youngs Modulus, E.

Reply:
I did it for the diff order, for this one, we'll first check if we can get this to linear elastic behavior